lib/rubocop/cop/github/rails_link_href.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require "rubocop"
+
+module RuboCop
+  module Cop
+    module GitHub
+      module Accessibility
+        class LinkHref < Base
```

Comment:
Maybe we should rename this to `LinkHasHref` for clarity!

Reply:
Sounds reasonable